pkg/daemon/ceph/osd/volume.go

```diff
@@ -283,7 +281,12 @@ func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *Device
 				deviceArg = device.Config.Name
 			}
 
-			immediateExecuteArgs := append(baseArgs, []string{
+			crushDeviceClass := os.Getenv(oposd.CrushDeviceClassVarName)
```

Comment:
Do this after the first initialization of `immediateExecuteArgs` otherwise this is breaking the CLI.